tools/db_bench_tool.cc

```diff
@@ -605,8 +606,24 @@ static enum ROCKSDB_NAMESPACE::CompressionType
         ROCKSDB_NAMESPACE::kLZ4Compression;
 
 DEFINE_string(compression_manager, "none",
-              "Set the compression manager type to mixed(roundrobin) or other "
-              "type. None for BuilInCompressor");
+              "Set the compression manager type to mixed(roundrobin), "
+              "autoskip, autotunecompressor, "
+              "or none. Use 'none' for BuiltInCompressor");
+
+DEFINE_double(autotune_io_upper_bound, 0.99,
+              "Ratio of rate_limiter budget to set as IO goal for autotune "
+              "compression manager");
+DEFINE_double(
+    autotune_io_lower_bound, 0.9,
+    "Ratio of rate_limiter budget to set as minimum IO goal for autotune "
+    "compression manager");
```

Comment:
In db bench we set it to ratio to rate limiter. In the public API it's absolute bytes "between maximum and minimum bytes per second". Should we consolidate?